pkg/awx/connection.go

```diff
@@ -322,18 +324,19 @@ func (c *Connection) rawPost(path string, query url.Values, input []byte) (outpu
 	c.setCredentials(request)
 	c.setContentType(request)
 	c.setAccept(request)
+	if glog.V(2) {
+		glog.Infof("Sending POST request to '%s'.", address)
+		dump, err := httputil.DumpRequest(request, true)
```

Comment:
Maybe there is a reason to prefer `c.indent(input)` to log request body?

Reply:
The only reason is that the output will be indented, which simplifies reading for humans. I guess that `DumpRequest` doesn't do that. Not a big problem. Can you include in the description of the pull request an example of how the dumped information looks like?